cmd/crio/main.go

```diff
@@ -353,6 +360,10 @@ func main() {
 			logrus.Fatalf("failed to listen: %v", err)
 		}
 
+		if len(config.Registries) == 0 {
+			logrus.Fatal("no registries configured. Please add registries with the --registry flag")
```

Comment:
I'm torn between leaving this error or do as you said, force people to use fully qualify images. However, I think we should go with the latter (error out) so we don't risk to break kubernetes at all (since many users still use unqualified images in prod)